packages/manager/modules/telecom-dashboard/src/identity-check/form/telecom-dashboard-identity-check-form.html

```diff
@@ -0,0 +1,151 @@
+<section class="telecom-legacy">
+    <tuc-toast-message></tuc-toast-message>
+    <div class="text-center" data-ng-if="$ctrl.isLoading">
+        <oui-spinner></oui-spinner>
+    </div>
+    <div data-ng-if="!$ctrl.isLoading">
+        <div class="telecom-dashboard">
+            <header class="oui-header mb-4">
+                <div class="oui-header__container">
+                    <div class="oui-header__content">
+                        <h1
+                            class="oui-header__title"
+                            data-translate="telecom_dashboard_identity_check_form_title"
+                        ></h1>
+                        <h2
+                            class="oui-header__description"
+                            data-translate="telecom_dashboard_identity_check_form_description"
+                            data-ng-if="!$ctrl.procedure"
+                        ></h2>
+                    </div>
+                </div>
+            </header>
+        </div>
+        <oui-message data-type="info" data-ng-if="$ctrl.procedure">
+            <p
+                data-translate="telecom_dashboard_identity_check_form_procedure_info"
+            ></p>
+            <p>
+                <a
+                    class="oui-link_icon"
+                    href="{{ $ctrl.procedure.pdfUrl }}"
+                    target="_blank"
+                    rel="noopener noreferrer"
+                >
+                    <span
+                        data-translate="telecom_dashboard_identity_check_form_download_pdf"
+                    ></span>
+                    <span class="oui-icon oui-icon-arrow-right"></span>
+                </a>
+            </p>
+            <p>
+                <a
+                    class="oui-link_icon"
+                    data-ng-click="$ctrl.cancelProcedure()"
+                    href
```

Comment:
Should it done with a `oui-button` (or `button` if some style issues occurs). Also, you could remove the test to cancel the method logic here: https://github.com/ovh/manager/pull/6017/files#diff-d742e5c71cde846ceabb73e1a73cc3cb07fe078b1fa8c4bb774486ad5a6e2e8cR73

Reply:
> Should it done with a oui-button (or button if some style issues occurs)

I really don't know :confused:  
Currently on the mock up this is a simple link as you can see
![image](https://user-images.githubusercontent.com/3984480/143551593-29ae75ef-92e9-4906-8292-20b0173281f9.png)
Should I recommend using a design system button instead ?

Reply:
Done :+1: